### ISSUE TITLE
How to rescale the normalized prediction from NHITS model ?

### ISSUE THREAD

**Issue by Hvsk22** [jdb78/pytorch-forecasting#1307]
- PyTorch-Forecasting version: '1.0.0'
- PyTorch version: '2.0.0+cu117'
- Python version: '3.5.9'
- Operating System: 'unix'

### Expected behavior

I expected the NHITS model to predict the raw values when I call predict method on NHITS. 

### Actual behavior

However, the result was an array of standardized values that were not on the scale required. 

### Code to reproduce the problem

```

early_stop_callback = EarlyStopping(monitor="val_loss", min_delta=1e-4, patience=10, verbose=False, mode="min")
trainer = pl.Trainer(
    max_epochs=5,
    accelerator="cpu",
    enable_model_summary=True,
    gradient_clip_val=1.0,
    callbacks=[early_stop_callback],
    limit_train_batches=30,
    enable_checkpointing=True,
)


net = NHiTS.from_dataset(
    training,
    learning_rate=res.suggestion(),
    log_interval=10,
    log_val_interval=1,
    weight_decay=1e-2,
    backcast_loss_ratio=0.0,
    hidden_size=64,
    optimizer="AdamW",
    loss=MQF2DistributionLoss(prediction_length=13),
)

trainer.fit(
    net,
    train_dataloaders=train_dataloader,
    val_dataloaders=val_dataloader,
)

best_model_path = trainer.checkpoint_callback.best_model_path
best_model = NHiTS.load_from_checkpoint(best_model_path)
raw_predictions = best_model.predict(val_dataloader, mode="raw", return_index = True,return_x=True,return_decoder_lengths=True)

raw_predictions.output.prediction.numpy()
```

Paste the command(s) you ran and the output. Including a link to a colab notebook will speed up issue resolution.
If there was a crash, please include the traceback here.
The code used to initialize the TimeSeriesDataSet and model should be also included.

```
training_cutoff = train_data['time_idx'].max() - start_validation_weeks
validation_cutoff = training_cutoff + validation_weeks

# for all the categorical feature NaNlabelEncoder is used to encode
categorical_list = []
time_varying_known_categorical_list = ['brand', 'country', 'dept_id', 'div_id']
categorical_encoders = {
    col: NaNLabelEncoder(add_nan=True)
    for col in categorical_list + time_varying_known_categorical_list
}

# COMMAND ----------

training = TimeSeriesDataSet(
    train_data[lambda x: x.time_idx <= training_cutoff],
    time_idx='time_idx',
    target=target[0],
    group_ids=group_ids,
    categorical_encoders=categorical_encoders,
    time_varying_known_categoricals=time_varying_known_categorical_list,
    # target normalization happens at group level because there are multiple groups in the dataset.
    target_normalizer=GroupNormalizer(
        groups=group_ids,
        method=method,
        transformation='log1p',
        method_kwargs=method_kwargs,
    ),
    weight=None,
    min_encoder_length=min_input_len,
    max_encoder_length=max_input_len,
    min_prediction_length=min_output_len,
    max_prediction_length=max_output_len,
    time_varying_unknown_reals=['value'],
    add_relative_time_idx=False,
    add_target_scales=False,
    add_encoder_length='auto'
)

# COMMAND ----------

train_dataloader = training.to_dataloader(
        train=True,
        batch_size=batch_size,
        num_workers=num_workers,
        pin_memory=True,
        persistent_workers=True
    )

# COMMAND ----------

validation = TimeSeriesDataSet.from_dataset(
    training,
    train_data[lambda x: x.time_idx <= validation_cutoff],
    min_prediction_idx=training_cutoff,
    stop_randomization=True
)
val_dataloader = validation.to_dataloader(
    train=False,
    batch_size=batch_size,
    num_workers=num_workers,
    pin_memory=True,
    persistent_workers=True
)

# COMMAND ----------

prediction_dataset = TimeSeriesDataSet.from_dataset(
    training,
    test_data,
    predict=True,
    stop_randomization=True,
    min_prediction_idx=validation_cutoff +1 
)
predict_dataloader = prediction_dataset.to_dataloader(
    train=False,
    batch_size=batch_size * 10,
    num_workers=num_workers
)
```
